pynamodb/expressions/__init__.py

```diff
@@ -0,0 +1,35 @@
+import re
```

Comment:
i generally dislike code in the `__init__` files. garrett (aka 🐍  man) might think it's pythonic or something tho